packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -48,13 +49,17 @@ class AppSizeController {
   /// The node set as the analysis tab root.
   ///
   /// Used to build the treemap and the tree table for the analysis tab.
-  ValueListenable<TreemapNode?> get analysisRoot => _analysisRoot;
-  final _analysisRoot = ValueNotifier<TreemapNode?>(null);
+  final analysisRoot =
+      ValueNotifier<Selection<TreemapNode>>(Selection<TreemapNode>());
+
+  ValueListenable<int> get selectedNodeIndex => _selectedNodeIndex;
+  final _selectedNodeIndex = ValueNotifier<int>(0);
 
   void changeAnalysisRoot(TreemapNode? newRoot) {
-    _analysisRoot.value = newRoot;
     if (newRoot == null) return;
```

Comment:
We should probably do the same thing we are doing on line 160 if `newRoot` is null (set the analysisRoot to be an empty selection)